Tests/ConfigurationTests/EnvironmentVariablesProviderTests.swift

```diff
@@ -124,6 +125,45 @@ struct EnvironmentVariablesProviderTests {
         #expect(throws: ConfigError.self) { try sut.value(forKey: "BOOLY_ARRAY_THROWS_1", type: .boolArray) }
         #expect(throws: ConfigError.self) { try sut.value(forKey: "BOOLY_ARRAY_THROWS_2", type: .boolArray) }
         #expect(throws: ConfigError.self) { try sut.value(forKey: "BOOLY_ARRAY_THROWS_3", type: .boolArray) }
+        #expect(throws: ConfigError.self) { try sut.value(forKey: "BOOLY_ARRAY_THROWS_4", type: .boolArray) }
+    }
+
+    @available(Configuration 1.0, *)
+    @Test func valueForKeyOfIntArrayTypes() throws {
```

Comment:
Please add some success cases as well here for the ints and doubles

Reply:
Done.